src/main/java/io/github/haykam821/clutchpractice/clutch/PlacementClutchType.java

```diff
@@ -7,28 +7,40 @@
 import io.github.haykam821.clutchpractice.game.map.ClutchPracticeMap;
 import net.minecraft.block.Block;
 import net.minecraft.block.BlockState;
+import net.minecraft.fluid.Fluid;
 import net.minecraft.item.ItemStack;
 import net.minecraft.server.world.ServerWorld;
 import net.minecraft.text.Text;
 import xyz.nucleoid.plasmid.util.ItemStackBuilder;
 
 public class PlacementClutchType extends ClutchType {
```

Comment:
Would it be possible to instead split this class into a `BlockPlacementClutchType` based on the block name and item and `FluidPlacementClutchType` based on the bucket name and item? Alternatively, I believe blocks and fluids can be ignored in place of only using item stacks.

Reply:
Going to go with the latter, as I don't see any reason why we need the separation when ItemStacks can accomplish the same while being more universal

Reply:
New commit fixes this